conf/sample/nsc-memif.yaml

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: nsm-client-init
+  name: nsm-client-init
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app: nsm-client-init
+  name: nsm-client-init
+  namespace: default
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - pods
+    verbs:
+      - get
+      - list
+  - apiGroups:
+      - networkservicemesh.io
+    resources:
+      - networkserviceendpoints
+    verbs:
+      - get
+      - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    app: nsm-client-init
+  name: nsm-client-init
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: nsm-client-init
+subjects:
+  - kind: ServiceAccount
+    name: nsm-client-init
+---
+apiVersion: v1
+data:
+  networkService: |
+    - name: gold-network
+      serviceInterface:
+      - type: 2
+        preference: 3
+        metadata:
+          name: interface-3
+      - type: 3
+        preference: 2
+        metadata:
+          name: interface-2
+      - type: 1
+        preference: 1
+        metadata:
+          name: interface-1
+kind: ConfigMap
+metadata:
+  name: client-network-services
+  namespace: default
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        networkservicemesh.io: "true"
+        networkservicemesh.io/app: nsm-client
+    spec:
+      serviceAccount: nsm-client-init
+      containers:
+        - name: nsm-init
+          image: networkservicemesh/nsc-memif
+          imagePullPolicy: IfNotPresent
+          args:
+            - --nsm-socket=/var/lib/networkservicemesh/nsm.ligato.io.sock
+            - --configmap-name=client-network-services
+          resources:
+            limits:
+              nsm.ligato.io/socket: 1
+          env:
+            - name: INIT_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+      volumes:
+        - hostPath:
+            path: /var/lib/networkservicemesh
```

Comment:
We will need to get per-Pod directories injected into NSCs and NSEs rather than mounting the whole NSM directory to ensure proper isolation and security.